doc/source/admin/security.md

```diff
@@ -14,7 +18,7 @@ Note that a tool only knows the paths to its inputs and outputs, but if using th
 
 There are three approaches to protect Galaxy against these risks:
 
-- Use different credentials for running tools. This can be configured using the ``real_system_username`` config variable.
+- Use different credentials for running tools and make ``file_path``, ``tool_data_path`` writable only by the user running Galaxy. This can be configured using the ``real_system_username`` config variable. Note that this implies ``outputs_to_working_directory``.
```

Comment:
I don't think that generally applies, and on the surface the better solution is to use `outputs_to_working_directory` with a container destination, in which case those directories are mounted ro within jobs.

Reply:
Still I think that this is worth documenting for instances that can not use containers. Maybe we should switch the order and highlight the best solution.